ocs_ci/ocs/cluster.py

```diff
@@ -833,6 +833,37 @@ def validate_ocs_pods_on_pvc(pods, pvc_names, pvc_label=None):
             ), f"No PVC {pvc_name} found for pod: {pod_name} in PVCs: {pvc_names}!"
 
 
+@retry(CommandFailed, tries=3, delay=10, backoff=1)
+def validate_claim_name_match_pvc(pvc_names, validated_pods=None):
+    """
+    Validate if OCS pods have mathching PVC and Claim name
+
+    Args:
+        pvc_names (list): names of all PVCs you would like to validate with.
+        validated_pods(set): set to store already validated pods - if you pass
+            empty set outside of this function, this will speed up the next
+            validation when re-tries as it will skip those already validated
+            pods stored in this set.
```

Comment:
Wouldn't be this slightly more clear?
```suggestion
            pods added to this set by previous run of this function.
```